programs/drift/src/ids.rs

```diff
@@ -6,11 +7,21 @@ pub mod pyth_program {
     declare_id!("gSbePebfvPy7tRqimPoVecS2UsBvYv46ynrzWocc92s");
 }
 
+pub mod wormhole_program {
+    use solana_program::declare_id;
+    declare_id!("HDwcJBJXjL9FpJ7UBsYBtaDjsBUhuLCUYoz3zr8SWWaQ");
+}
+
 pub mod pyth_pull_program {
```

Comment:
can remove this one?

Reply:
no i  use it when checking encoded_vaa account owner